addon/components/menu.hbs

```diff
@@ -1,6 +1,7 @@
 <div
   id={{this.guid}}
   {{on 'pointerdown' this.onPointerDown}}
+  {{click-outside this.close}}
```

Comment:
@achambers fyi, I also needed your modifier from https://github.com/GavinJoyce/ember-headlessui/pull/31 here

Reply:
Coolio. I'll change mine to use that.